src/partials/how-we-do.html

```diff
@@ -0,0 +1,118 @@
+    <section class="how-we-do__section">
+      <div class="container">
+        <h1 class="how-we-do__heading">How do we do it</h1>
+        <ul class="how-we-do__list">
+          <li class="how-we-do__item">
+            <div class="how-we-do__svg-list">
+              <svg class="how-we-do__svg">
+                <use href="/images/sprite.svg#icon-study-program"></use>
+              </svg>
+            </div>
+            <ul class="how-we-do__text">
```

Comment:
Lista z jednym elementem nie jest potrzebna. Można zamienić na < div >, element < li < z wewnątrz po prostu usunąć. Stylowanie z tego < li > (padding) można przenieść do diva how-we-do__text